client/src/stores/ModalComponentStore.js

```diff
@@ -1,6 +1,7 @@
 export const createModalComponentStore = () => {
   return {
     isOpen: false,
+    isClosable: true,
```

Comment:
I like this, it prevents users from accidentally cancelling their match requests